io-uring-test/src/tests/net.rs

```diff
@@ -1436,6 +1437,163 @@ pub fn test_udp_recvmsg_multishot<S: squeue::EntryMarker, C: cqueue::EntryMarker
 
     Ok(())
 }
+pub fn test_udp_recvmsg_multishot_trunc<S: squeue::EntryMarker, C: cqueue::EntryMarker>(
+    ring: &mut IoUring<S, C>,
+    test: &Test,
+) -> anyhow::Result<()> {
+    require!(
+        test;
+        test.probe.is_supported(opcode::RecvMsgMulti::CODE);
+        test.probe.is_supported(opcode::ProvideBuffers::CODE);
+        test.probe.is_supported(opcode::SendMsg::CODE);
+    );
+
+    // Check kernel version
+    {
+        let mut uname: libc::utsname = unsafe { std::mem::zeroed() };
+        unsafe {
+            assert!(libc::uname(&mut uname) >= 0);
+        }
+
+        match uname
+            .release
```

Comment:
we can introduce a crate to parse kernel version number.

Reply:
Done.